### PR TITLE
Fixed Slack samples.

### PR DESCRIPTION
**Pull request by danepowell**
The Slack samples are woefully out of date and need some TLC. Specifically, Slack has updated its webhook API and no longer uses tokens or requires explicitly defining a channel in the message payload.

Also, this adds a check if the slack settings file exists before attempting to include it. Otherwise, deployment tasks fail if the file doesn't exist.

### CODE REVIEW

File: samples/slack/slack_settings [hunk 1]
```diff
@@ -1,2 +1 @@
-TOKEN=yourtoken
-SLACK_WEBHOOK_URL=https://yoursite.slack.com/services/hooks/incoming-webhook?token=$TOKEN
\ No newline at end of file
+SLACK_WEBHOOK_URL=https://hooks.slack.com/services/xxx/yyy/zzz
```
Comment (ElijahLynn) [inline samples/slack/slack_settings]: Will see about testing this out soon. Until then, this file needs an ending newline. Thanks for this!

**Comment by ElijahLynn** [comment]
Cross referencing #43

**Comment by ksenzee** [comment]
Thanks for this update! I'd reroll if Github had a workflow that allowed such things. Alas. Meanwhile, in addition to the newline, it needs s/cody/code/.

**Comment by danepowell** [comment]
@ksenzee or @ElijahLynn any chance at a merge? We are starting to use this downstream and it's going to confuse folks if we have mismatched docs: https://github.com/acquia/blt/pull/1000

Thanks!

**Comment by ElijahLynn** [comment]
I haven't had a chance to try this. But looks decent enough from me. Better than what exists right now.

**Comment by ksenzee** [comment]
I can't merge (not at Acquia anymore!) but I've tested it and it works for me.

**Comment by hoff331** [comment]
pinging some project contributors who still work at Acquia for review and (possible) merger: @codexmas , @marksakurada , @grasmash

**Pull request merged**